azure-pipelines.yml

```
@@ -97,7 +97,7 @@ steps:
     GIT_BRANCH: $(Build.SourceBranchName)
 - script: |
     set -x
-    $DOCKER_OVERRIDE exec -T web /bin/sh -c "bundle exec rake cc:report"
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c "export GIT_BRANCH=${GIT_BRANCH} && bundle exec rake cc:report"
```

```suggestion
    $DOCKER_OVERRIDE exec -T -e GIT_BRANCH=$GIT_BRANCH web rake cc:report
```

I'm just testing this, but:

- the `-e` flag is meant to do just this
- `exec` is documented as taking the args: `COMMAND [ARGS ...]`
- rake doesn't need `bundle exec` I'm pretty sure anymore (I'm still testing)